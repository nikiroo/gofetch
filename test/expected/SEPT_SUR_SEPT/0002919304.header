0Le Titan de Vorombe, le plus gros oiseau que l'Homme ait connu	null/SEPT_SUR_SEPT/0002919304		70
i  [Espèces Menacées] 26-09-2018 16:08
i
i    Le  plus  gros  oiseau du monde serait bien un oiseaux-éléphant
i    mais  pas celui qu'on croyait: après plus d'un siècle de pièces
i    contradictoires ...
i
