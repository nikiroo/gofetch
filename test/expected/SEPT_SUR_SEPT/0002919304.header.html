<!DOCTYPE html>
<html>
<head>
  <meta http-equiv='content-type' content='text/html; charset=utf-8'>
  <meta name='viewport' content='width=device-width, initial-scale=1.0'>
  <style type='text/css'>
    body { margin: 1em 15%; }
  </style>
</head>
<body>
<div class='story-header'>
	<h2><a href='0002919304.html'>Le Titan de Vorombe, le plus gros oiseau que l'Homme ait connu</a></h2>
	<div class='details'>([Espèces Menacées] 26-09-2018 16:08)</div>
	<br/>
	<div class='content' style='text-align: justify'>
		Le plus gros oiseau du monde serait bien un oiseaux-éléphant mais pas celui qu&#x27;on croyait: après plus d&#x27;un siècle de pièces contradictoires ...
	</div>
<hr/>
</div>
</body>
